dags/ods/kktix_ticket_orders/kktix_dag.py

```diff
@@ -34,11 +34,7 @@
         provide_context=True,
     )
 
-    SEND_MSG_TO_DISCORD = PythonOperator(
-        task_id="LOAD_TO_DISCORD", python_callable=discord_bot.send,
-    )
-
-    CREATE_TABLE_IF_NEEDED >> GET_ATTENDEE_INFOS >> SEND_MSG_TO_DISCORD
```

Comment:
moved `SEND_MSG_TO_DISCORD` task to a dedicated DAG, with its own cadence (daily)